src/middleware/auth/user/authDataOnlyMiddleware.ts

```diff
@@ -12,18 +9,14 @@ const authDataOnlyMiddleware = async (req: Request, res: Response, next: Next) =
 
   if (userAccessToken == null) return res.sendStatus(401);
```

Comment:
To retrieve the token you can use https://www.npmjs.com/package/express-bearer-token